lib/message_bus/rails/generators/migrations/message_bus_migration_generator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails/generators/migration'
+require 'message_bus/rails/models/message'
+
+module MessageBus
+  class MessageBusMigrationGenerator < ::Rails::Generators::Base
+    include ::Rails::Generators::Migration
+
+    source_root File.expand_path('templates', __dir__)
+    desc "Create migrations for ActiveRecord adapter of MessageBus"
+
+    # @param migrations_root [String]
+    # @return [String]
+    def self.next_migration_number(migrations_root)
```

Comment:
I think you can use ActiveRecord implementation. So don't need to implement yourself this `next_migration_number`:
https://apidock.com/rails/ActiveRecord/Generators/Migration/next_migration_number

Reply:
Like suggested here: 
https://stackoverflow.com/a/38542994

Reply:
That is old implementation. New implementation accepts `current_migration_number` as an argument. Refs https://api.rubyonrails.org/classes/ActiveRecord/Migration.html#method-i-next_migration_number